docs/CompatibleSoftware.rst

```diff
@@ -238,50 +388,16 @@ profile (it will be selected for you by default). This is because ICC Profile
 are not LUTs per se. An ICC Profile describes a color space and then needs a
 destination profile to calculate the transformation. So if you have an operation
 working and looking good on the monitor you’re using (and maybe its
-profile has been properly measured using a spectrophotometer), then choose your
-display. If the transform was approved on a different monitor, then maybe you
 should choose its profile instead.
 
 
-RV
-*********
-
-`Playback Tool - Tweak Software <http://www.tweaksoftware.com>`__
+Compatible Software (Deprecated)
+--------------------------------
 
-RV has native OCIO support in version 4 onwards. For more details, see
-the OpenColorIO section of the `RV User Manual
-<http://www.tweaksoftware.com/static/documentation/rv/current/html/rv_manual.html#OpenColorIO>`__.
+Hiero (Foundry)
```

Comment:
Aparantly I was wrong about Hiero being totally deprecated (now that I go digging around). It is a part of Nuke Studio, but they are still marketing it as a product, so this can be moved back up in the main section. Please also update with:

```
Hiero ships with native support for OCIO in the display and the equivalent of Nuke's OCIOColorSpace in the Read nodes. It comes with the "nuke-default" OCIO config by default, so the Hiero viewer
matches when sending files to Nuke for rendering.

Website: `<https://www.foundry.com/products/hiero>`__

Supported version: >= 1.0
```